### ISSUE TITLE
Upload latest version in RubyGem

### ISSUE THREAD

**Issue by equintana** [typus/typus#504]
Our team want to use typus on a gem were are building but the last version in ruby gem is 3.1.10 which isn't compatible with Rails 4.2, Do you have plan to upload it?

**Comment by fesplugas** [comment]
Code in **master** branch is Rails 4.2 compatible, but there are some parts which are broken and I don't when I'll have time to fix them, so this is what blocks me from pushing a new version of the ruby gem.

**Comment by equintana** [comment]
Yes, I know the code in master is working, we did some test in a rails application, but got a blocker when we tried to add the gem dependency in the .gemspec file, since it doesn't accepts github as repository, it only looks for gems in gems' servers, like Ruby gems.
Thanks for reply, we will be pending any update.
